ios/FluentUI/Bottom Sheet/BottomSheetController.swift

```diff
@@ -9,7 +9,10 @@ import UIKit
 public protocol BottomSheetControllerDelegate: AnyObject {
 
     /// Called after the sheet moved to a new expansion state
-    @objc optional func bottomSheetControllerDidMove(to expansionState: BottomSheetExpansionState)
+    @objc optional func bottomSheetController(_ bottomSheetController: BottomSheetController, didMoveTo expansionState: BottomSheetExpansionState)
+
+    /// Called when `collapsedSheetHeight` changes.
```

Comment:
Probably more to add in the comments here about the subtleties of how this behaves in expanded and hidden states?

Reply:
Added more documentation.